.gitignore

```diff
@@ -135,3 +135,5 @@ simgui.json
 simgui-ds.json
 # Simulation GUI and other tools window save file
 *-window.json
+#ignore vscode files
+.vscode/
```

Comment:
We shouldn't actually gitignore the entirety of `.vscode/`, since we actually have some of that config checked in.

What did we want to ignore here?

Reply:
I think it was `settings.json` that got modified.